test/server/config_validation/xds_fuzz.h

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include <memory>
+#include <string>
+
+#include "envoy/admin/v3/config_dump.pb.h"
+#include "envoy/config/cluster/v3/cluster.pb.h"
+#include "envoy/config/endpoint/v3/endpoint.pb.h"
+#include "envoy/config/listener/v3/listener.pb.h"
+#include "envoy/config/route/v3/route.pb.h"
+
+#include "test/common/grpc/grpc_client_integration.h"
+#include "test/config/utility.h"
+#include "test/integration/http_integration.h"
+#include "test/server/config_validation/xds_fuzz.pb.h"
+
+#include "absl/types/optional.h"
+
+// TODO(samflattery): add these to fuzz config instead?
+#define NUM_LISTENERS 3
```

Comment:
See bullet point about constant vars here:
https://github.com/envoyproxy/envoy/blob/master/STYLE.md

Maybe prefer against #define and use static const?

Reply:
Should that be a global or should I just make it a member variable of the class?

Reply:
Prefer latter then? Some of the fuzzing constants just exist as class members
https://github.com/envoyproxy/envoy/blob/347ec29ab1834e14b49df4e24618ee07d57cf3c2/test/common/http/header_map_impl_fuzz_test.cc#L23